### PR TITLE
Correct binding of invalid property

### PR DESCRIPTION
**Pull request by lluissuros**
* Correct binding of invalid property
 * Add tests

Solves https://github.com/PolymerElements/gold-cc-cvc-input/issues/52

### CODE REVIEW

**Comment by lluissuros** [comment]
I signed it!

File: gold-cc-cvc-input.html [hunk 1]
```diff
@@ -160,6 +160,11 @@
           value: {
             type: String,
             observer: '_onValueChanged'
+          },
+
+          invalid: {
```
Comment (notwaldorf) [inline gold-cc-cvc-input.html]: This _should_ come from `PaperInputBehavior`, where it's already `notify: true:` :https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L81

**Comment by notwaldorf** [comment]
I think this is the right solution, thanks for tracking it down! I don't think redefining the `invalid` property is needed, tho :)

File: gold-cc-cvc-input.html (hunk 1 above)
Comment (lluissuros) [inline gold-cc-cvc-input.html]: next commit thx

**Comment by lluissuros** [comment]
@notwaldorf thx for your comments, reviewed

**Comment by mcrowder65** [comment]
Thank you very much for this pull request, just discovered this bug a couple of minutes ago.

**Comment by notwaldorf** [comment]
Thanks!

**Pull request merged**